build/ci/evergreen.yml

```diff
@@ -1248,6 +1248,27 @@ tasks:
           AZURE_CLIENT_ID: ${logs_decrypt_azure_client_id}
           AZURE_CLIENT_SECRET: ${logs_decrypt_azure_client_secret}
           E2E_TAGS: atlas,decrypt
+  # export buckets require cloud provider role authentication and a s3 bucket created
+  - name: atlas_backups_export_buckets_e2e
+    tags: [ "e2e","buckets","atlas" ]
+    must_have_test_results: true
+    depends_on:
+      - name: compile
+        variant: "code_health"
+        patch_optional: true
+    commands:
+      - func: "clone"
+      - func: "install gotestsum"
+      - func: "e2e test"
+        vars:
+          MCLI_ORG_ID: ${atlas_org_id}
+          MCLI_PROJECT_ID: ${atlas_project_id}
+          MCLI_PRIVATE_API_KEY: ${atlas_private_api_key}
+          MCLI_PUBLIC_API_KEY: ${atlas_public_api_key}
+          MCLI_OPS_MANAGER_URL: ${mcli_ops_manager_url}
+          MCLI_SERVICE: cloud
+          E2E_TAGS: atlas,buckets
+          E2E_CLOUD_ROLE_ID: ${e2e_cloud_role_id}
```

Comment:
you're missing `E2E_TEST_BUCKET`

Reply:
You're right. I changed it at the end from the constant and of course forget to add the variable here. Thanks.